_events/sr2019/competition.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Competition
+date: 2019-04-06 9:00:00
```

Comment:
We should probably specify that this lasts over both days.

Reply:
Not sure how to do this in a way which will display on the site. I'm basing it off the 2017 event which used the single date: https://www.studentrobotics.org/events/sr2017/competition/
(Although it did have a schedule for clarity. Perhaps we should have day 1 and day 2 events separately?)

Reply:
This is now mentioned inline; unfortunately the limitations of the front-matter parsing mean it can only handle a single datetime.